tasks.yaml

```diff
@@ -138,6 +133,15 @@ tasks:
   - name: build-deploy-custom-slim
     description: "Build/Deploy custom slim dev bundle for integration testing"
     actions:
+      - description: "Replace dev versions in the dev bundle with the VERSION variable"
+        cmd: |
+          VERSION=$(yq eval '.metadata.version' uds-core/bundles/k3d-standard/uds-bundle.yaml)
```

Comment:
nit: we should use `uds zarf tools yq` here to make this more portable

Reply:
Done